include/EVT/io/platform/f4xx/UARTf4xx.hpp

```diff
@@ -0,0 +1,57 @@
+#ifndef _EVT_UARTf3xx_
+#define _EVT_UARTf3xx_
+
+#include <stdint.h>
+
+#include <EVT/io/UART.hpp>
+
+#include <HALf4/stm32f4xx.h>
+
+namespace EVT::core::IO {
+
+class UARTf4xx : public UART {
+public:
+    /**
+     * Create an instance of the STMF302x8 UART interface using the provided
+     * TX and RX pins.
+     *
+     * @param[in] txPin The UART TX pin.
+     * @param[in] rxPin THe UART RX pin.
+     * @param[in] baudrate The baudrate to operate the UART with/
+     */
+    UARTf4xx(Pin txPin, Pin rxPin, uint32_t baudrate);
```

Comment:
Why did you get rid of isSwapped?

Reply:
You probably based this on an old version of UARTf3xx.hpp. Just look at the new version (after the big merge you did) and make it look like that